app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class VotesController < ApplicationController
+  def create
+
+    if !current_user?
+      flash[:alert] = 'Please sign in to vote'
+      redirect_to auth_sign_in_path
```

Comment:
A better way to do this: https://github.com/chaynHQ/soulmedicine/blob/469974625dbf828ad85dc332fe306b508eb708c9/app/controllers/subscriptions_controller.rb#L2

Reply:
Done